<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>128</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>128</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="systemPanel"/>
    <widget class="QWidget" name="mpxPanel">
     <widget class="QLabel" name="labelType">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Type-034</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInfo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mpx Info</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMpx1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>34.8 dBu</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMpx2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>20.9 dBu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rdsPanel">
     <widget class="QLabel" name="labelRdsLevel">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>45</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>01234</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroup1">
      <property name="geometry">
       <rect>
        <x>191</x>
        <y>21</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10A</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroup2">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>21</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10A</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroup3">
      <property name="geometry">
       <rect>
        <x>287</x>
        <y>21</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10A</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroup4">
      <property name="geometry">
       <rect>
        <x>191</x>
        <y>45</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10A</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroup5">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>45</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10A</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroup6">
      <property name="geometry">
       <rect>
        <x>287</x>
        <y>45</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10A</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPTYN">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>83</y>
        <width>110</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>OPERETTA</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPTY">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>108</y>
        <width>110</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>CLASSICA</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDS">
      <property name="geometry">
       <rect>
        <x>347</x>
        <y>45</y>
        <width>12</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPS">
      <property name="geometry">
       <rect>
        <x>347</x>
        <y>21</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>RADIO101</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPI">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>45</y>
        <width>40</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>5024</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDI">
      <property name="geometry">
       <rect>
        <x>429</x>
        <y>45</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>ST</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRT">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>83</y>
        <width>270</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>MICHAEL JACKSON - Bad</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRdsPhase">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>21</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>149.7</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="audioPanel"/>
    <widget class="QWidget" name="mpxSpectrumPanel"/>
    <widget class="QWidget" name="audioSpectrumPanel"/>
    <widget class="QWidget" name="splashPanel"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
